.changelog/release-v15.0.0.md

```diff
@@ -0,0 +1,174 @@
+14. operator-settlement _(DEPRECATED)_ - https://github.com/mojaloop/operator-settlement/releases/tag/v11.0.0
+15. event-sidecar - https://github.com/mojaloop/event-sidecar/releases/tag/v12.0.0
+16. event-stream-processor - https://github.com/mojaloop/event-stream-processor/releases/v12.0.0-snapshot.7
+17. simulator - https://github.com/mojaloop/simulator/releases/tag/v12.0.0
+18. mojaloop-simulator - https://github.com/mojaloop/mojaloop-simulator/releases/tag/v13.0.1
+19. sdk-scheme-adapter - https://github.com/mojaloop/sdk-scheme-adapter/releases/tag/v21.4.0
+20. ml-testing-toolkit - https://github.com/mojaloop/ml-testing-toolkit/releases/tag/v15.2.0
+21. ml-testing-toolkit-ui - https://github.com/mojaloop/ml-testing-toolkit-ui/releases/tag/v15.0.1
+22. ml-testing-toolkit-client-lib - https://github.com/mojaloop/ml-testing-toolkit-client-lib/releases/tag/v1.0.0
+23. auth-service - https://github.com/mojaloop/auth-service/releases/tag/v13.0.2
+24. als-consent-oracle - https://github.com/mojaloop/als-consent-oracle/releases/tag/v0.2.0
+25. thirdparty-api-svc - https://github.com/mojaloop/thirdparty-api-svc/releases/tag/v13.0.2
+26. thirdparty-sdk-adapter - https://github.com/mojaloop/thirdparty-sdk/releases/tag/v15.1.0
+
+## 5. Breaking changes
+
+1. ....
+
+## 6. Deprecations
+
+The following components have been deprecated from the packaged Mojaloop Helm chart release and have been with [Business Operation Framework (BOF)](https://github.com/mojaloop/business-operations-framework-docs), which can be deployed by the offical [BoF Helm Chart](https://github.com/mojaloop/charts/tree/master/mojaloop/bof):
+
+- [finance-portal](https://github.com/mojaloop/helm/tree/master/finance-portal) Helm Chart
+  - [finance-portal-ui](https://github.com/mojaloop/finance-portal-ui/releases/tag/v10.4.3)
+  - [finance-portal-backend-service](https://github.com/mojaloop/finance-portal-backend-service/releases/tag/v15.0.2)
+- [finance-portal-settlement-management](https://github.com/mojaloop/helm/tree/master/finance-portal-settlement-management) Helm Chart
+  - [settlement-management](https://github.com/mojaloop/settlement-management/releases/tag/v11.0.0)
+  - [operator-settlement](https://github.com/mojaloop/operator-settlement/releases/tag/v11.0.0)
+
+> _Note these Helm Chart are still available for deployment in the [Mojaloop Helm Repo (http://mojaloop.io/helm/repo/)](http://mojaloop.io/helm/repo/index.yaml)._
+
+This is due to the underlying services having been deprecated by the Micro-Services provided by the [Business Operation Framework (BOF)](https://github.com/mojaloop/business-operations-framework-docs) for Financial Management and Reporting.
+
+More information can be found here:
+
+- https://github.com/mojaloop/business-operations-framework-docs.
+
+[BOF Helm charts](https://github.com/mojaloop/charts/tree/master/mojaloop/bof) to deploy the [Business Operation Framework](https://github.com/mojaloop/business-operations-framework-docs) can be found here:
```

Comment:
I went to both the links indicated here and I could not find anything to say how to install the BOF

Reply:
That would a great contribution for the BoF and its adoption.

@PaulGregoryBaker @elnyry-sam-k FYI.